mlrun/runtimes/sparkjob/spark3job.py

```diff
@@ -175,6 +185,11 @@ def _enrich_job(self, job):
             update_in(
                 job, "spec.executor.nodeSelector", self.spec.executor_node_selector
             )
+        if self.spec.driver_tolerations:
+            update_in(job, "spec.driver.tolerations", self.spec.driver_tolerations)
+        if self.spec.executor_tolerations:
+            update_in(job, "spec.executor.tolerations", self.spec.executor_tolerations)
```

Comment:
So you added support setting `.spec.tolerations` for spark but IIUC you never look on it ?
that doesn't make much sense, we need to think whether we really need to support it

Reply:
you right I missed handling executor_tolerations and driver_tolerations, btw maybe we should let users set tolerations with `with_node_selection` and populate the values to executor and driver ( or this could result unexpected behavior for the user that we prefer to restrict)?